middleware/recover.go

```diff
@@ -38,6 +38,10 @@ type (
 		// LogErrorFunc defines a function for custom logging in the middleware.
 		// If it's set you don't need to provide LogLevel for config.
 		LogErrorFunc LogErrorFunc
+
+		// DisableErrorHandler disables the call to centralized HTTPErrorHandler.
```

Comment:
and it would be best here to mention that - in that case we return that recovered error ( instead of "swallowing" it). Our new feature/behavior.